### ISSUE TITLE
Kerberos leak open files

### ISSUE THREAD

**Issue by JumpingYang001** [Microsoft/omi#350]
Run Kerberos command:
```
/opt/omi/bin/omicli --hostname ub16-02 --auth Kerberos -u kdcaccount@CONTOSO.COM -p password--port 1270 ei root/scx SCX_UnixProcess --encryption https
```

Check omiserver open files:(23029 is pid for omiserver process)
```
ls -l /proc/23029/fd
```

The open files increase and doesn't close once you run the Kerberos command each time.

**Comment by yakman2020** [comment]
This was due to the failure to call gss_delete_sec_context on disconnect. 

Fixed in commit 9ed6742fc9c016582bd81a6b3535a85b75ed01cf

**Comment by yakman2020** [comment]
Confirmed fixed by jumping
